public/amavis-recent-mail.php

```diff
@@ -25,17 +25,17 @@
 
```

Comment:
DATE_SUB is missing from Postgresql, investigating other ways to specify the where condition

Reply:
I hope the change in the placement of the quoting works the same like it tested i this [sqlfiddle](http://sqlfiddle.com/#!15/7c7b8/1/0)